ha/const.py

```diff
@@ -23,6 +23,10 @@
 HACLUSTER_KEY = "cortx"
 SERVER_NODE_KEY = "server_node"
 RA_LOG_DIR="/var/log/seagate/cortx/ha"
+RA_LOG_FILES = [f"{RA_LOG_DIR}/alert_monitor.log", f"{RA_LOG_DIR}/cluster_manager.log",
+                f"{RA_LOG_DIR}/event_analyzerd.log", f"{RA_LOG_DIR}/event_manager.log",
+                f"{RA_LOG_DIR}/health_monitor.log"]
+
```

Comment:
This will not scale in case more log files get added

Reply:
I think we can add modules log file names here in the list